changelog.d/611.change.rst

```diff
@@ -0,0 +1 @@
+Fixed serialization of classes with `frozen=True` and `cache_hash=True`, which has been broken since 19.1.0.
```

Comment:
This ain't Markdown, Paul. 😇

Reply:
This is what I get for making a PR when I'm in a rush to get out the door. Good catch, thanks!